packages/smooth_app/lib/pages/onboarding/knowledge_panel_page_template.dart

```diff
@@ -55,71 +59,73 @@ class _KnowledgePanelPageTemplateState
   }
 
   @override
-  Widget build(BuildContext context) {
-    final MaterialColor materialColor = SmoothTheme.getMaterialColor(context);
-    return FutureBuilder<void>(
-      future: _initFuture,
-      builder: (BuildContext context, AsyncSnapshot<void> snapshot) {
-        if (snapshot.hasError) {
-          final AppLocalizations appLocalizations =
-              AppLocalizations.of(context);
-          return Text(
-            appLocalizations.knowledge_panel_page_loading_error(snapshot.error),
-          );
-        }
-        if (snapshot.connectionState != ConnectionState.done) {
-          return const Center(child: CircularProgressIndicator());
-        }
-        final Widget knowledgePanelWidget =
-            const KnowledgePanelsBuilder().buildSingle(
-          _knowledgePanels,
-          widget.panelId,
-          context: context,
-        )!;
-        return Scaffold(
-          body: Stack(
-            fit: StackFit.expand,
-            children: <Widget>[
-              ListView(
-                // bottom padding is very large because [NextButton] is stacked on top of the page.
-                padding: const EdgeInsets.only(
-                  top: LARGE_SPACE,
-                  right: LARGE_SPACE,
-                  left: LARGE_SPACE,
-                  bottom: VERY_LARGE_SPACE * 5,
-                ),
-                shrinkWrap: true,
+  Widget build(BuildContext context) => FutureBuilder<void>(
+        future: _initFuture,
+        builder: (BuildContext context, AsyncSnapshot<void> snapshot) {
+          if (snapshot.hasError) {
+            final AppLocalizations appLocalizations =
+                AppLocalizations.of(context);
+            return Text(
+              appLocalizations
+                  .knowledge_panel_page_loading_error(snapshot.error),
+            );
+          }
+          if (snapshot.connectionState != ConnectionState.done) {
+            return const Center(child: CircularProgressIndicator());
+          }
+          final Widget knowledgePanelWidget =
+              const KnowledgePanelsBuilder().buildSingle(
+            _knowledgePanels,
+            widget.panelId,
+            context: context,
+          )!;
+          return Container(
+            color: widget.backgroundColor,
+            child: SafeArea(
+              child: Stack(
+                fit: StackFit.expand,
                 children: <Widget>[
-                  Padding(
-                    padding: const EdgeInsets.symmetric(
-                      horizontal: LARGE_SPACE,
+                  ListView(
+                    // bottom padding is very large because [NextButton] is stacked on top of the page.
```

Comment:
I know it's not from you but if there is some easy fix for just not showing content behind the nextButton that be great

Reply:
Probably just a `Column` with a scrolling content on top and a bar on bottom.
But that's not a priority.